daemon/daemon.go

```diff
@@ -539,7 +539,6 @@ func (d *Daemon) installIptablesRules() error {
 	// with a destination IP in the cluster range.
 	if err := runProg("iptables", []string{
 		"-A", ciliumForwardChain,
-		"-d", node.GetIPv4ClusterRange().String(),
 		"-o", "cilium_host",
```

Comment:
Wouldn't `GetIPv4AllocRange` go here, like the rule below?

Reply:
Unfortunately, that will be too restrictive as a local process talking to a remote pod IP will go this path and will need to be masqueraded. As written in the commit message, it is safe to ignore the destination IP as the path into cilium_host can only be taken if a route is pointing to it. I've clarified this with a better comment.